modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -7212,6 +7212,11 @@ type Flamingos2Dynamic {
   Flamingos 2 reads.
   """
   reads: Flamingos2Reads!
+
+  """
+  Central wavelength, which is taken from the filter wavelength.
```

Comment:
is this for one step? l'm thinking of imaging where we have n filters

Reply:
Correct, it’s the instrument config for one particular step. If there are steps with different filters they would have corresponding different wavelengths.